gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -1072,23 +1074,34 @@ message Stop {
   extensions 9000 to 9999;
 }
 
+// NOTE: This field is still experimental, and subject to change. It may be formally adopted in the future.
 message TripModifications {
 	message Modification {
-    // The first stop sequence of the original trip that is to be affected by this modification.
-		optional uint32 start_stop_sequence = 1;
-
-    // The number of stops which are canceled and replaced by the replacement_stops. May be zero to indicate no skipped stops.
-		optional uint32 num_stops_replaced = 2;
-
-    // The number of seconds of delay to add to all departure and arrival times following the end of this modification. If multiple modifications apply to the same trip, the delays accumulate as the trip advances. 
+    // he stop selector of the first stop of the original trip that is to be affected by this modification. 
+    // If `end_stop_selector` is not provided, no stop will be removed and the stop from the `start_stop_selector` 
+    // is only provided as a reference stop for `travel_time_to_stop` calculations.
+		optional StopSelector start_stop_selector = 1;
+
+    // The stop selector of the last stop of the original trip that is to be affected by this modification. 
+    // The selection is inclusive, so if only one stop_time is replaced by that modification, `start_stop_selector` and `end_stop_selector` should be equivalent.
+    // If no stop_time is replaced, `end_stop_selector` should not be provided. It's otherwise required.
+		optional StopSelector end_stop_selector = 2;
```

Comment:
If no stop is replaced, I understand you don't provide a end_stop_selector. If you provide a start_stop_selector in this case, it is to modify the travel time between the start_stop and the start_stop + 1?

Reply:
Yes exactly

Reply:
Do you have language improvement you want to suggest or it's understandable enough like this?

Reply:
Okay, I had to read it multiple times. Maybe adding something like the above to make it more explicit?